install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,602 @@ $ oc label node --all logging-infra-fluentd=true <1>
+|Time when the message was received at the collector.
+
+| `pipeline_metadata.normalizer.original_raw_message`
+|The original non-parsed log message as it is received at the normalizer.
+
+| `pipeline_metadata.trace`
+|The field records the trace of the message. Each collector/normalizer appends
+information about itself and the date/time when the message was processed.
+|===
+
+[discrete]
+[[exported-fields-ovirt]]
+=== oVirt Fields
+
+Namespace for oVirt metadata.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `ovirt.entity`
+|The type of the data source, hosts, VMS, and engine.
+
+| `ovirt.host_id`
+|The oVirt host UUID.
+|===
+
+[discrete]
+[[exported-fields-ovirt.engine]]
+=== `ovirt.engine` Fields
+
+Namespace for oVirt engine related metadata. The FQDN of the oVirt engine is
+`ovirt.engine.fqdn`
+
+[discrete]
+[[exported-fields-aushape]]
+=== Aushape Fields
+
+Audit events converted with Aushape. For more information please see
+link:https://github.com/Scribery/aushape[Aushape].
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `aushape.serial`
+|Audit event serial number.
+
+| `aushape.node`
+|Name of the host where the audit event occurred.
+
+| `aushape.error`
+|The error aushape encountered while converting the event.
+
+| `aushape.trimmed`
+|An array of JSONPath expressions relative to the event object, specifying
+objects/arrays with content removed as the result of event size
+limiting. Empty string means event itself. Empty array means trimming occurred
+at unspecified objects/arrays.
+
+| `aushape.text`
+|An array log record strings representing the original audit event.
+|===
+
+[discrete]
+[[exported-fields-aushape.data]]
+=== `aushape.data` Fields
+
+Parsed audit event data related to Aushape.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `aushape.data.avc`
+|type: nested
+
+| `aushape.data.execve`
+|type: string
+
+| `aushape.data.netfilter_cfg`
+|type: nested
+
+| `aushape.data.obj_pid`
+|type: nested
+
+| `aushape.data.path`
+|type: nested
+|===
+
+[discrete]
+[[exported-fields-tlog]]
+=== Tlog Fields
+
+Tlog terminal I/O recording messages. For more information please see
+link:https://github.com/Scribery/tlog[Tlog].
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `tlog.ver`
+|Message format version number.
+
+| `tlog.user`
+|Recorded user name.
+
+| `tlog.term`
+|Terminal type name.
+
+| `tlog.session`
+|Audit session ID of the recorded session.
+
+| `tlog.id`
+|ID of the message within the session.
+
+| `tlog.pos`
+|Message position in the session, milliseconds.
+
+| `tlog.timing`
+|Distribution of this message's events in time.
+
+| `tlog.in_txt`
+|Input text with invalid characters scrubbed.
+
+| `tlog.in_bin`
+|Scrubbed invalid input characters as bytes.
+
+| `tlog.out_txt`
+|Output text with invalid characters scrubbed.
+
+| `tlog.out_bin`
+|Scrubbed invalid output characters as bytes.
+|===
+endif::openshift-origin[]
```

Comment:
@richm 
I see the following data fields,  shall we need to add them?
collectd.interval
collectd.plugin
collectd.plugin_instance
collectd.type_instance
collectd.type
collectd.dstypes
collectd.processes.ps_state
collectd.processes.ps_disk_ops.read
collectd.processes.ps_disk_ops.write
collectd.processes.ps_vm
collectd.processes.ps_rss
collectd.processes.ps_data
collectd.processes.ps_code
collectd.processes.ps_stacksize
collectd.processes.ps_cputime.user
collectd.processes.ps_cputime.syst
collectd.processes.ps_count.processes
collectd.processes.ps_count.threads
collectd.processes.ps_pagefaults.majflt
collectd.processes.ps_pagefaults.minflt
collectd.processes.ps_disk_octets.read
collectd.processes.ps_disk_octets.write
collectd.processes.fork_rate
collectd.disk.disk_merged.read
collectd.disk.disk_merged.write
collectd.disk.disk_octets.read
collectd.disk.disk_octets.write
collectd.disk.disk_time.read
collectd.disk.disk_time.write
collectd.disk.disk_ops.read
collectd.disk.disk_ops.write
collectd.disk.pending_operations
collectd.disk.disk_io_time.io_time
collectd.disk.disk_io_time.weighted_io_time
collectd.interface.if_octets.rx
collectd.interface.if_octets.tx
collectd.interface.if_packets.rx
collectd.interface.if_packets.tx
collectd.interface.if_errors.rx
collectd.interface.if_errors.tx
collectd.interface.if_dropped.rx
collectd.interface.if_dropped.tx
collectd.virt.if_octets.rx
collectd.virt.if_octets.tx
collectd.virt.if_packets.rx
collectd.virt.if_packets.tx
collectd.virt.if_errors.rx
collectd.virt.if_errors.tx
collectd.virt.if_dropped.rx
collectd.virt.if_dropped.tx
collectd.virt.disk_ops.read
collectd.virt.disk_ops.write
collectd.virt.disk_octets.read
collectd.virt.disk_octets.write
collectd.virt.memory
collectd.virt.virt_vcpu
collectd.virt.virt_cpu_total
collectd.CPU.percent
collectd.df.df_complex
collectd.df.percent_bytes
collectd.entropy.entropy
collectd.nfs.nfs_procedure
collectd.memory.memory
collectd.memory.percent
collectd.swap.swap
collectd.swap.swap_io
collectd.load.load.shortterm
collectd.load.load.midterm
collectd.load.load.longterm
collectd.aggregation.percent
collectd.statsd.host_cpu
collectd.statsd.host_elapsed_time
collectd.statsd.host_memory
collectd.statsd.host_nic_speed
collectd.statsd.host_nic_rx
collectd.statsd.host_nic_tx
collectd.statsd.host_nic_rx_dropped
collectd.statsd.host_nic_tx_dropped
collectd.statsd.host_nic_rx_errors
collectd.statsd.host_nic_tx_errors
collectd.statsd.host_storage
collectd.statsd.host_swap
collectd.statsd.host_vdsm
collectd.statsd.host_vms
collectd.statsd.vm_nic_tx_dropped
collectd.statsd.vm_nic_rx_bytes
collectd.statsd.vm_nic_tx_bytes
collectd.statsd.vm_balloon_min
collectd.statsd.vm_balloon_max
collectd.statsd.vm_balloon_target
collectd.statsd.vm_balloon_cur
collectd.statsd.vm_cpu_sys
collectd.statsd.vm_cpu_usage
collectd.statsd.vm_disk_read_ops
collectd.statsd.vm_disk_write_ops
collectd.statsd.vm_disk_flush_latency
collectd.statsd.vm_disk_apparent_size
collectd.statsd.vm_disk_write_bytes
collectd.statsd.vm_disk_write_rate
collectd.statsd.vm_disk_true_size
collectd.statsd.vm_disk_read_rate
collectd.statsd.vm_disk_write_latency
collectd.statsd.vm_disk_read_latency
collectd.statsd.vm_disk_read_bytes
collectd.statsd.vm_nic_rx_dropped
collectd.statsd.vm_cpu_user
collectd.statsd.vm_nic_rx_errors
collectd.statsd.vm_nic_tx_errors
collectd.statsd.vm_nic_speed
collectd.postgresql.pg_n_tup_g
collectd.postgresql.pg_n_tup_c
collectd.postgresql.pg_numbackends
collectd.postgresql.pg_xact
collectd.postgresql.pg_db_size
collectd.postgresql.pg_blks